charts/capsule/Chart.yaml

```diff
@@ -21,7 +21,7 @@ sources:
 
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
-version: 0.3.5
+version: 0.4.0
```

Comment:
My personal experience: adding values doesn't require a minor bump, a patch is good enough.

We're using minor changes when updating the CRD definition, and major when changing or deprecating Helm values.

Please, let's coordinate with #718 to get it merged, and then bump up to the next patch version.